feincms_extensions/render_json.py

```diff
@@ -0,0 +1,18 @@
+from feincms import extensions
+
+
+class Extension(extensions.Extension):
+    def handle_model(self):
+        cls = self.model
+
+        def render_json(self, request):
+            """Render the feincms regions into a dictionary."""
+
+            def region_data(region):
+                content_list = getattr(self.content, region.key)
+                return [content.json(request=request) for content in content_list]
```

Comment:
Should this check that `content.json` exists / is available? I am not sure what the correct behaviour is if it does not but it might be useful to have an explicit error as it is not `standard` FeinCMS behaviour and potentially undocumented .